src/components/Popup.js

```diff
@@ -137,6 +138,7 @@ function Title(props) {
     children,
     className,
     draggable,
+    emit,
```

Comment:
Nitt-pick: We can do null-object-pattern once again here:

```suggestion
    emit || () => {},
```

Reply:
```suggestion
    emit = () => {},
```
;)

Reply:
💌